.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Test'
+
+on: [push]
+
+jobs:
+  make:
+    name: 'Make'
+    runs-on: ubuntu-latest
+    env:
+      # For storing the Terraform state for the deployment
+      AWS_ACCESS_KEY_ID: "${{ secrets.AWS_ACCESS_KEY_ID }}"
+      AWS_SECRET_ACCESS_KEY: "${{ secrets.AWS_SECRET_ACCESS_KEY }}"
+
+    # Use the Bash shell regardless whether the GitHub Actions runner is ubuntu-latest, macos-latest, or windows-latest
+    defaults:
+      run:
+        shell: bash
+
+    # Checkout the repository to the GitHub Actions runner
+    steps:
+    - name: Install the eden OSBAPI CLI tool
+      run: |
+        wget -q -O - https://raw.githubusercontent.com/starkandwayne/homebrew-cf/master/public.key | sudo apt-key add -
+        echo "deb http://apt.starkandwayne.com stable main" | sudo tee /etc/apt/sources.list.d/starkandwayne.list
+        sudo apt-get update
+        sudo apt-get install eden
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: '0'
+
+    - name: Supply an empty .env.secrets file
+      run: |
+        echo "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" > .env.secrets
+        echo "AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY" >> .env.secrets
+    - name: Build
+      run: make build
+
+    - name: Set the instance name for subsequent steps
+      # Use an instance name that corresponds to this particular branch+run.
+      # Docs for this method:
+      # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
+      run: |
+        INSTANCE_NAME="$(echo ci-${{ github.event.pull_request.number }}-${GITHUB_RUN_ID})"
+        echo "INSTANCE_NAME=${INSTANCE_NAME}" | tee -a $GITHUB_ENV
```

Comment:
This env var is used in the Makefile for the eks-brokerpak and datagov-brokerpak, but it has no purpose here, so these lines can be removed.